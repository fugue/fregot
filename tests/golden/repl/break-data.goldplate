{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file", "step.rego"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        ":break data.step.test_step",
        "data.step.test_step",
        ":continue"
    ]
}
