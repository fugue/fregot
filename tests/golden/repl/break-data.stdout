repl% repl% 14|   low_a
      ^^^^^
step(debug)% (debug) = true
(debug) finished
repl% 